networkit/viztasks.py

```diff
@@ -30,8 +58,18 @@ def coloringToColorList(G, coloring):
 
 
 def drawGraph(G, **kwargs):
-	""" Draws a graph via networkX. Passes additional arguments beyond the graph to networkx.draw(...).
```

Comment:
Since we have our own cytoscape-based tool, we don't need to rely on networkx' graph drawing features anymore. This can be deprecated as well, I think.